src/utils/hederaUtils.js

```diff
@@ -156,24 +145,35 @@ module.exports = class HederaUtils {
           new HederaSDK.Hbar(-10000)
         );
       const tx = await transferTransaction.execute(client);
-      await tx.getReceipt(client);
-
-      const accountInfo = await new HederaSDK.AccountInfoQuery({
-        accountId: HederaSDK.AccountId.fromEvmAddress(0, 0, wallet.address),
-      }).execute(client);
+      if (startup) {
+        logger.log(
```

Comment:
Could move log out of if else and instead assign the value to an accountId variable.
Then after the if else
```suggestion
        logger.log(
```
```suggestion
        let accountId;
        if () {
          ...
          accountId = `0.0.${firstAccountNum}`
          ...
        } else {
          ...
          accountId = accountInfo.accountId.toString()
          ...
        }
        
        logger.log(
          `| ${accountId} - ${wallet.address} - ${
            wallet._signingKey().privateKey
          } - ${new HederaSDK.Hbar(10000)} |`
        );
```

Reply:
Thank you, i've extracted this logic in separate function.